lib/handlers/router-entities/route-caching/cached-routes-configuration.ts

```diff
@@ -159,9 +159,12 @@ export const CACHED_ROUTES_CONFIGURATION: Map<string, CachedRoutesStrategy> = ne
       tradeType: TradeType.EXACT_INPUT,
       chainId: ChainId.MAINNET,
       buckets: [
-        new CachedRoutesBucket({ bucket: 100, cacheMode: CacheMode.Tapcompare, withLastNCachedRoutes: 10 }),
```

Comment:
Let's instead try to just Darkmode this bucket for 100 and see how things change